src/lucky/routeable.cr

```diff
@@ -27,26 +40,71 @@ module Lucky::Routeable
     end
   end
 
+  # Define a nested route that responds to the appropriate HTTP request
+  # automatically
+  #
+  # This works similarly to `action` but it will provide multiple parameters.
+  # For example:
+  # ```
+  # class Posts::Comments::Show
+  #   nested_action do
+  #     render_text "Post: #{post_id}, Comment: #{id}"
+  #   end
+  # end
+  # ```
+  #
+  # **Note:** The `singular` option will likely be removed soon. Try `get`,
+  # `post`, `put`, and `delete` with a custom path instead.
   macro nested_action(singular = false)
     infer_nested_route(singular: {{ singular }})
 
     setup_call_method({{ yield }})
   end
 
+  # Define a route that responds to the appropriate HTTP request automatically
+  #
+  # A route needs a few pieces of information to be created:
+  #
+  # * The HTTP method, like `GET`, `POST`, `DELETE`, etc.
+  # * The path, such as `/users/:id`
+  # * The class to route to, like `Users::Show`
+  #
+  # The `action` method will try to determine these pieces of information based
+  # the class name. After it knows the class, Lucky will transform the full
+  # class name to figure out the path, i.e. removing the `::` separators and
+  # adding underscores. The method is found via the last part of the class name:
+  #
+  # * `Index` -> `GET`
+  # * `Show` -> `GET`
+  # * `New` -> `GET`
+  # * `Create` -> `POST`
+  # * `Edit` -> `GET`
+  # * `Update` -> `PUT`
+  # * `Delete` -> `DELETE`
```

Comment:
It would be good to link here as well: https://luckyframework.org/guides/actions-and-routing/#automatically-generate-restful-routes

And/or use the list here: https://luckyframework.org/guides/actions-and-routing/#examples-of-automatically-generated-routes